test/vcr_python_wrapper.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import vcr
+import json
+
+VCR_PARAMS_FILE = '../../.vcr_params.json'
```

Comment:
I'm thinking about passing this path as an environment variable and using ```mktemp```. Maybe we'll be able to run things in parallel at some point and then it's useful.

Reply:
Have a try! :)
I think the environment get's cleared by ansible. Otherwise, we could pass a lot of interesting information that way.

Reply:
Looks like that works: https://github.com/ekohl/foreman-ansible-modules/commit/30386617792f5d022c5c00ad572006588f0778cc

Reply:
Seems i have been mistaken about the environment. 🎆